### PR TITLE
Feat/37 artist toast

### PR DESCRIPTION
**Pull request by katjam**
Partial #37

- Adds hardcoded instructions speech bubble & artist above card chooser

NOTE: see #37 for next steps planned

### CODE REVIEW

File: elm.json [hunk 1]
```diff
@@ -13,9 +13,11 @@
             "elm/random": "1.0.0",
             "elm/url": "1.0.0",
             "elm-community/random-extra": "3.2.0",
-            "fapian/elm-html-aria": "1.4.0"
+            "fapian/elm-html-aria": "1.4.0",
+            "pablen/toasty": "1.2.0"
```
Comment (Nikomus) [inline elm.json:17]: What does this do?

File: src/Main.elm [hunk 2]
```diff
@@ -346,23 +372,12 @@ renderGameArea model =
...
-              else
-                div [ class "help-text" ]
-                    [ p [] [ text model.cardSetMeta.help ]
-                    , p [] [ text "Why not challenge yourself to guess it right first time?" ]
-                    ]
-            ]
+renderArtistSpeech : String -> Html Msg
+renderArtistSpeech speech =
+    div [ class "artist-speech container" ]
+        [ img [ class "artist", src "card-images/barbara.svg" ] []
```
Comment (Nikomus) [inline src/Main.elm:378]: Missing alt, should probably convey that the text is from a person.

**Review by Nikomus — APPROVED** [review]
Looks nice so far!

File: src/Main.elm (hunk 2 above)
Comment (katjam) [reply src/Main.elm:378]: Good point. I'll add.

File: elm.json (hunk 1 above)
Comment (katjam) [reply elm.json:17]: Not a lot. I might end up getting rid of it.

**Pull request merged**